lib/commands/exam.js

```diff
@@ -20,7 +20,7 @@ module.exports = TestCommand.extend({
 
   availableOptions: [
     {
-      name: 'split',
+      name: 'partition-count',
```

Comment:
Although this is more accurate, is the cost of `split` sufficiently high to require a rename? I worry the churn is more costly then the benefit. Thoughts?

Reply:
I personally have to do mental gymnastics to disambiguate the options in their current state. I _always_ convert `split` to `partition count` in my mind. I'm in favor of naming it what it is. 

I think we could keep both for backwards compatibility, but I'd just opt for a full conversion over with a major bump. 

Am open to other thoughts.